packages/cosmwasm-stargate/src/signingcosmwasmclient.ts

```diff
@@ -201,11 +201,9 @@ export class SigningCosmWasmClient extends CosmWasmClient {
     options: SigningCosmWasmClientOptions,
   ) {
     super(tmClient);
-    // TODO: do we really want to set a default here? Ideally we could get it from the signer such that users only have to set it once.
-    const prefix = options.prefix ?? "cosmos";
     const {
       registry = createDefaultRegistry(),
-      aminoTypes = new AminoTypes({ prefix, additions: createWasmAminoConverters() }),
+      aminoTypes = new AminoTypes({ ...createWasmAminoConverters() }),
```

Comment:
Here we need a few other modules as well. At least bank for `sendTokens`. But maybe we should better keep all the standard types that were available before.